src/renderer/views/create-torrent-error-page.js

```diff
@@ -0,0 +1,26 @@
+const React = require('react')
+
+const {dispatcher} = require('../lib/dispatcher')
```

Comment:
Is this dispatcher library still the best way to accomplish this? Doesn't react offer something cleaner?